source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -1171,6 +1171,12 @@ bool ContextImpl::verifyCertChain(X509& leaf_cert, STACK_OF(X509) & intermediate
     error_details = "Failed to verify certificate chain: X509_STORE_CTX_init";
     return false;
   }
+  // Currently only EnvoyQuicProofVerifier, which is used by the client code, calls this method. So
+  // hard-code "ssl_server" for now.
```

Comment:
I think that since this method is always called to verify a chain of certs presented by a server, it is fine to use ssl_server here. In other words I don't think we need to mention EnvoyQuicProofVerifier in the comment and instead can simply explain that this method is always call to verify server certs.

What do you think?

Reply:
I rephrased the comment.